servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1436,7 +1438,7 @@ public boolean mount(TachyonURI tachyonPath, TachyonURI ufsPath)
         return true;
       }
     }
-    return false;
+    throw new IOException("Internal error when mounting.");
```

Comment:
should mountInternal be throwing exception instead of returning a boolean to provide for more information error message?